pkg/yamlmeta/document_set.go

```diff
@@ -9,14 +9,14 @@ import (
 )
 
 type DocSetOpts struct {
-	WithoutMeta bool
-	Strict      bool
+	WithoutComment bool
```

Comment:
WithoutComment -> WithoutComments since we are dealing with multiple.

Reply:
👍